apps/keyboard/test/unit/keyboard/user_press_manager_test.js

```diff
@@ -1114,4 +1116,177 @@ suite('UserPressManager', function() {
       manager.stop();
     });
   });
+
+  suite('speedlimit', function() {
+    var manager, el, dummyKey, clock;
+
+    setup(function() {
+      clock = sinon.useFakeTimers();
```

Comment:
Yes it affects the Date object, we need it to control the startTime and endTime values in the UserPress.